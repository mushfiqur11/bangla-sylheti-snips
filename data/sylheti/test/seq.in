ব্যাটারি পার্ক ও যাওয়ার লাগি কুনো বালা ফরামরশ জানার আছে নি?
ঔ শাফতাত সিলেট ক্লাব ও কিতা ওর?
আমারে অ্যাম্বারখানা তাকি জিন্দাবাজার যাওয়ার রাস্তা দেও
৩ দিন বাদর আবহাওয়া কিলান?
শুক্রবার রাইত আমার হোটেল ও শবতাকি জলদি জাওয়া জায় অলান রাস্তা দেখাও
আইজ রাইতর লাগি চিকেন চাপ আর দুধ সা ফাওা জায় অলান রেস্টুরেন্ট ও ৩ জন মাইনশর লাগি এখটা টেবিল বুক খড়
দুফুর ওর দিকে স্টেক খাওয়ার লাগি রেল স্টেশন ওর খানদাত এখটা জাগা দরখার
ই শাফতার শেষে আমার Airbnb ও weather কিলান অইব?
আইজ রাইত ৬ জনর লাগি উন্দাল ও এখটা টেবিল বুক খড়
ফাহিম ওর লগে আমার ETA শেয়ার কখোরোইন
রাইত ৮ টার লাগি কেবাবিশ রেস্টুরেন্ট ও ৮ জনর এখটা টেবিল বুক খড়
আমার হোটেল ওর খানদাত সবচে বালা জাদুগর ওগুইন দেখাও
পানশিত এখন ভিড় কিলা?
আমরা ৪ জনর group আর আমরা Arcadia ও শইন্দার সময় এখটা টেবিল বুক খরতাম ছাই
বাইরে কিতা ঠাণ্ডা নি?
খাইল আমার সারাদিনর অনুষ্ঠানর খান্দাত কিতা বিষটি অইব নি?
আমি স্কুলও ফউসানি ফরজন্ত আমার মার লগে আমার location hare খড়িও
ওউ শুক্রবার রাইত ৮ টার দিকে Domino ও ৪ জনর লাগি এখটা টেবিল বুক খড়
আমার facebook event ওর ইনো কিতা মেগলা থাখব নি?
আমার এখনু এখটা ট্যাক্সি দরখার
খাইল শোখালে আবহাওয়া কিলান অইব?
আমারে জিন্দাবাজার ওর সবচে বড় স্বপ্নর ঠিকানা দেও
আমারে খান্দাখাসাত নিরামিষ রেস্টুরেন্ট দেখাও
আমারে স্কাইডাইন ওর কিছু সবি দেখাও
সিলেট জেলা স্টেডিয়াম ও আইজ খে খেলের?
লতিফ ওর লগে আমার location সবসময় hare খড়িও
আমার বাফ-মার ইনো আবহাওয়া কিলান?
আমারে নূরজাহান হোটেল ওর ঠিকানা দেও
আমার location আমার জামাইর খাসে ফাটাও
আমি সাস্ট ও কনসার্ট ওর টিকিট খই ফাইমু?
টোল অ্যাভয়েড খড়িয়া হরিপুর কিলা যাইতাম?
আমি জমশেদ ওর লগে যে coffee খাওয়ার জাগাত দেখা খররাম ইনো কিতা wifi আছে নি?
ফুটবল দলর লোগে আমার ফউসানির সময় শেয়ার খড়
