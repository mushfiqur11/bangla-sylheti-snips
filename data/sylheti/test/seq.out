B-place I-place O O O O O O O O O O O O O O O
O O O O B-place I-place O O O O O
O B-origin I-origin I-origin O O B-destination I-destination I-destination I-destination O O O
O O O O O O O O
O O O B-destination I-destination O O O O O O O O O O O O O
O O O O O O O O O O O O O O O O B-restaurant O B-partySize O O O O O O O O O O
O O O O O O O O B-restaurant I-restaurant O O O O O O O O O
O O O O B-location I-location I-location I-location I-location O O O O O O O
O O O B-partySize O O O B-restaurant I-restaurant I-restaurant O O O O O O
B-contact I-contact O O O O O O O O O O
O O B-partySize O O B-restaurant I-restaurant I-restaurant I-restaurant O O O O O O O O O
B-address I-address O O O O O O O B-category I-category O O O O
B-place I-place O O O O O
O B-partySize O O O O O B-restaurant I-restaurant I-restaurant O O O O O O O O O O O O O
B-location O O O O O
O O B-location I-location I-location I-location I-location O O O O O O O O O O O
O O O O O O O O O O B-contact O O O O O O O
O O O O O O O O O O O B-partySize O O O O O O O O
O B-location I-location I-location O O O O O O O O O O O O O
O O O O O B-rideOptionName O O
O O O O O O O O O O O
O B-place I-place I-place I-place O O O O O O O O
O O O O O O O B-specificItemSold I-specificItemSold I-specificItemSold B-category O
O B-place I-place I-place O O O O O
B-place I-place I-place O O O O O O
B-contact I-contact O O O O O O O O
B-location I-location I-location I-location I-location I-location I-location O O O O
O B-place I-place I-place O O O
O O B-contact I-contact I-contact O O O O
O B-place I-place I-place I-place I-place O O O O O O O
O O O O O O O B-destination I-destination O O O O O
O O O O O O O O O O O O O O O O O O O O O O O
B-contact I-contact I-contact O O O O O O O O O
